packages/calcite-components/src/tests/commonTests/t9n.ts

```diff
@@ -82,7 +82,7 @@ export async function t9n(componentTestSetup: ComponentTestSetup): Promise<void>
             return new Response(new Blob([JSON.stringify(fakeEsMessages, null, 2)], { type: "application/json" }));
           }
 
-          return orig.call(input, init);
+          return orig.call(window, input, init);
```

Comment:
First argument to `.call` is `this`.
Not sure why this worked before in Stencil, but this causes issues after migration.